Makefile

```diff
@@ -109,21 +109,21 @@ html-cover: test                                             ## Display coverage
 	@go tool cover -html=$(COVER_PROFILE)
 .PHONY: html-cover
 
-travisci-cover: html-cover                                   ## Test & generate coverage in the TravisCI format, fails unless executed from TravisCI
+ci-cover: html-cover                                         ## Test & generate coverage in the TravisCI format, fails unless executed from TravisCI
```

Comment:
Kept the reference to TravisCI in the comment, since goveralls does output in TravisCI format and it won't work unless run in TravisCI (where `COVERALLS_TOKEN` is defined).